cypress/TestCases/README.md

```diff
@@ -14,11 +14,9 @@ Given the environment has been set up for 'Accessibility' tests
 And '3rd party app' invokes the 'Firebolt' API to 'get closedCaptions settings'
 Then Firebolt platform responds for 'accessibility.closedCaptionsSettings' method and '<Method_JSON_Path>' is '<Content>'
 ```
-* [Location of glue](../../support/step_definitions/)
-* [Location of modules](../../fixtures/modules/)
-* [Location of fireboltCalls](../../fixtures/fireboltCalls/)
-* [Location of defaultTestData json](../../fixtures/defaultTestData.json)
-* [Location of cypress commands](../../support/cypress-commands)
+* [Location of glue](../support/step_definitions/)
+* [Location of fixtures](../fixtures/modules/)
+* [Location of cypress commands](../support/cypress-commands)
```

Comment:
link to cypress/fixtures/readme in line 18

Reply:
done